assets/js/vue/FileManager.vue

```diff
@@ -221,16 +221,11 @@ const getItems = (pathInfo) => new Promise((resolve, reject) => {
     `${Routing.generate('pelagos_api_get_files_dataset_submission')}/${datasetSubmissionId}?path=${pathInfo.path}`,
   ).then((response) => {
     resolve(response.data);
-    const filesTabValidator = document.getElementById('filesTabValidator');
+    const filesUploaded = document.getElementById('filesUploaded');
     const datasetFileTransferType = document.getElementById('datasetFileTransferType');
     if (response.data.length > 0) {
-      filesTabValidator.value = 'valid';
+      filesUploaded.value = 'valid';
       datasetFileTransferType.value = 'upload';
```

Comment:
We need to add the else case here, because when we upload a file it sets it to "valid" and then we delete it, it is still valid and you can submit the form without any files. I know this is an edge case, but we still need to set filesUploaded.value to null or empty string if response.data.length is 0